src/ElectricPokemon.java

```diff
@@ -0,0 +1,89 @@
+public class ElectricPokemon extends Pokemons {
+    //variabelen
+    boolean solar;
```

Comment:
Ik zou properties altijd private maken. Je wilt dat deze alleen kunnen worden gezet dmv een constructor of setter functies.